bundles/org.eclipse.pde.doc.user/whatsNew/pde_whatsnew.html

```diff
@@ -41,6 +41,87 @@ <h2>What's New in 4.27 (PDE)</h2>
   <col />
 </colgroup>
 <tbody>
+  <!-- ******************** Dialogs, Wizard and Views ********************** -->
```

Comment:
relevant topics should be un-commented (line 27-34) and sync ../topics_WhatsNew.xml with this.